docs/CIP/CIP-XXXX/README.md

```diff
@@ -0,0 +1,272 @@
+
+For an identified intersection, the server computes and publishes an ordered stream of events.
+This event stream provides a chronological record of on-chain events and affectively state changes for the wallet to derive its latest state.
+
+> [!NOTE]
+> The server sources client-relevant data through credential-dependent, point-specific database queries, though the implementation details are beyond this CIP's scope.
+
+![Sequence Diagram Synchronization](./images/02-Synchronization-Sequence.svg)
+
+#### Resumability
+
+Clients can resume synchronization from their last known `point`s after disconnecting. To minimize reconnection attempts, clients should provide a list of `point`s based on their most recent state, including some that may have been affected by rollbacks. This approach reduces the likelihood of the server failing to find a valid intersection, which would otherwise result in dropped connections and additional round trips.
+
+> [!NOTE]
+> More details are described in the [`subscribe`](./messages/client/subscribe.md) message.
+
+#### Rollback Handling
+
+In a [CAP](https://en.wikipedia.org/wiki/CAP_theorem) system like Cardano, which balances global consistency with availability, there is a challenge in dealing with rollback events that impact transaction finality. Regardless of whether one uses their own full node or a server provider for transaction submission - if such events are not properly handled with the help of monitoring the rolling window of [k blocks](https://plutus-apps.readthedocs.io/en/latest/plutus/explanations/rollback.html) and respective resubmission, there is no guarantee that the transaction is finalized, eg. becomes part of the immutable pefix of the Cardano blockchain.
+
+Since our wallet-optimized protocol defines that server-sent messages include the `point` message partial, the client can check if its last `point` is greater than any subsequent message `point` from the server, without the need to introduce a `rollback` specific message partial. This assumes that the client keeps a local copy of the mutuable suffix of the blockchain defined by the security parameter.
+
+#### Connection Management
+
+Clients are expected to reconnect on error, when providing invalid `point`s for which a server cannot find an intersection.
+Clients may send multiple [`subscribe`](./messages/client/subscribe.md) messages for different accounts, but subsequently take responsibility to coordinate server-sent events by time to their possibly differently synchronized accounts. When receiving a message that references a `point` that the first account has already seen, the client only applies it for its second account for example.
+
+### Versioning
+
+In our protocol, the client sends its list of supported versions as part of the initial [`subscribe`](./messages/client/subscribe.md) message. Upon receiving this information, the server selects the highest mutually supported version and associates it with the client.
```

Comment:
It would be best to define what is versioned before talking about how the negotiation takes place.